### DISCUSSION TITLE
Sum of object values

### DISCUSSION THREAD

**Discussion by prashasti-k** [josdejong/mathjs#3249] [Q&A] [answered]
I have an object with over 20 keys and each of the values is a number. I want an expression to return the sum of those values. In javascript we would do something like
`Object.values(c).reduce((sum, b) => sum + b, 0)`
How can I modify this to an expression?

**Comment by dvd101x** [comment, marked as answer]
I don't see a way to currently get all the values of an object, or convert an object to something that can be mapped.

Luckily you can import a function:
```js
math.import({ values: Object.values })
```
And then call your function.
```js
math.evaluate("sum(values({ a: 1, b: 2 }))")  // yields 3
```

**Reply by dvd101x** [reply]
Or as one line.
```js
math.evaluate('sum(values({a:1,b:2}))', { values: Object.values })
```
